mewarpx/mewarpx/diags_store/flux_diagnostic.py

```diff
@@ -679,8 +679,11 @@ def __init__(self, diag_steps, runinfo, overwrite=True,
 
         # if this run is from a restart, try to load flux data up to the
         # current point
-        if mwxrun.restart:
-            self._load_checkpoint_flux()
+        callbacks.installafterinit(
+            lambda _=None: self._load_checkpoint_flux() if mwxrun.restart else None
+        )
```

Comment:
```suggestion
        if mwxrun.restart:
            callbacks.installafterinit(self._load_checkpoint_flux)
```
Is there a reason this wouldn't work?

Reply:
`mwxrun.restart` is set in `mwxrun.init_run`

Reply:
ah right, I get it now. Comment would help :)